backend/inspirehep/migrator/tasks.py

```diff
@@ -200,6 +201,20 @@ def migrate_recids_from_mirror(
     return str(chord_task.id)
 
 
+@task_failure.connect
+def log_error(
+    task_id=None, exception=None, args=None, kwargs=None, traceback=None, einfo=None
+):
```

Comment:
you should add `*args, **kwargs` in case celery decides to add extra args in the future.